.expeditor/update_hab_version.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+# Distribute the version of Habitat, stored in .hab-version, across the components
+# in our projects.
+#
+set -evx
+
+hab_version=$(cat .hab-version)
+
+sed -i -r "s/bash -- -v .+$/bash -- -v $hab_version/" tools/hab.sh
+sed -i -r "s/export SUPPORTED_HAB_VERSION=\".+\"/export SUPPORTED_HAB_VERSION=\"$hab_version\"/" bin/studio-common
```

Comment:
Woow! I didn't know you could do this. 😅  - Why we don't just red the file?

Reply:
Both are valid approaches. I tend to take this approach because it prevents us from having to deal with having to know where that file is.

Reply:
This pattern also supports environments where you cant/shouldnt read the file during the runtime.

Reply:
@tduffield when does this runs? on merge or ... 🤔

Reply:
when a PR is merged. It is executed via Expeditor.